components/logWorkout/LogHeader.jsx

```diff
@@ -17,16 +18,26 @@ const StyledText = styled.Text`
   text-align: center;
 `;
 
-export default () => (
+const LogHeader = ({ type }) => (
   <Container>
     <StyledText style={{ width: 30 }}>Set</StyledText>
 
     <StyledText style={{ width: 70 }}>Previous</StyledText>
 
-    <StyledText style={{ width: 70 }}>lbs</StyledText>
+    {type === 'WEIGHT' ? <StyledText style={{ width: 70 }}>lbs</StyledText> : <StyledText style={{ width: 70 }}>Sec</StyledText> }
```

Comment:
Also small thing, but every header has first letter capitalized except for lbs. Maybe change to Lbs? Idk if that looks ugly lol

Reply:
Yeah Lbs kinda ugly, I thought it looked weird with lower case sec as well